tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/static_method_tests.bal

```diff
@@ -149,12 +149,19 @@ public function testGetDefaultValueWithBEnv() {
      test:assertEquals(defaultValue, 2021);
 }
 
-public function testDefaultDecimalArgs() {
-    decimal val = defaultDecimalArgs(5);
+public function testDefaultDecimalArgsAddition() {
+    decimal val = defaultDecimalArgsAddition(5);
     decimal expected = 15.05;
     test:assertEquals(val, expected);
 }
 
+public function testDefaultDecimalArgs() {
```

Comment:
Shall we add a test for all simple types with defaultable paramter in functions_with_default_parameters.bal

ex. funtion foo(string a = "eee", int b = 2, ....) {

int c = b + 6;// add some operation
}